src/backend/opencl/CMakeLists.txt

```diff
@@ -108,12 +108,22 @@ INCLUDE_DIRECTORIES(${CLFFT_INCLUDE_DIRS})
 LINK_DIRECTORIES(${CLFFT_LIBRARY_DIR})
 
 OPTION(USE_SYSTEM_BOOST_COMPUTE "Use system BoostCompute" OFF)
-IF(USE_SYSTEM_BOOST_COMPUTE)
-    IF(Boost_VERSION VERSION_LESS "1.61")
+
+ADD_DEFINITIONS(-DBOOST_ALL_NO_LIB)
+SET(Boost_USE_STATIC_LIBS OFF)
+FIND_PACKAGE(Boost REQUIRED)
+
```

Comment:
Boost.Compute is using Boost's platform specific headers(`config.hpp` which is not part of Compute), so we can't get rid of it whether the version of Boost is 1.61 or lesser.